src/web/mjs/connectors/Cmoa.mjs

```diff
@@ -0,0 +1,52 @@
+import SpeedBinb from './templates/SpeedBinb.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Cmoa extends SpeedBinb {
+
+    constructor() {
+        super();
+        super.id = 'cmoa';
+        super.label = 'コミックシーモア (Cmoa)';
+        this.tags = ['manga', 'japanese'];
+        this.url = 'https://www.cmoa.jp';
+    }
+
+    async _getMangaFromURI(uri) {
+        const request = new Request(uri, this.requestOptions);
+        const dom = await this.fetchDOM(request);
+        const id = dom.querySelector('#GA_this_page_title_id').textContent.trim();
+        const title = dom.querySelector('#GA_this_page_title_name').textContent.trim();
+        return new Manga(this, `/title/${id}/`, title);
+    }
+
+    async _getMangas() {
```

Comment:
Maybe scrape from https://www.cmoa.jp/search/result/?page=2495 but it has almost 2500 pages and may take very very long

Reply:
It is easy to fail load after 1500 pages and I feel like it will never end to list manga. I think it would be better not to implement it.
(Also, it seems like it would overload the site.)